.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
this file was auto generated by husky. i'm not entirely sure but i think it will find the first `sh`

itwinui-react has the same: https://github.com/iTwin/iTwinUI-react/blob/main/.husky/pre-commit